test12.md

```diff
@@ -0,0 +1,19 @@
+# Task 12 (WORK IN PROGRESS)
+### Responsive Layouts
+You watched course on Responsive Web Design in previous lessons. Time for practice! In this lesson you will use your knowledge to make a layout of an imaginary online pizza ordering service.
```

Comment:
I think it should be pizza order management service, because this tool looks more like a part of [POS](https://ru.wikipedia.org/wiki/POS-%D0%BE%D0%B1%D0%BE%D1%80%D1%83%D0%B4%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5) for staff usage.